src/osgPlugins/dae/daeReader.cpp

```diff
@@ -12,6 +12,9 @@
  */
 
 #include "daeReader.h"
+
+#include <cstddef>
```

Comment:
Why add this header?

I can't see anything in the new code that would require it.  I don't know whether it's inclusion would cause problems with very old compilers so unless it's really needed I wouldn't add it.

Reply:
It's for `std::size_t`, I think this header is very old, certainly in C++98

Reply:
Which is preferable then guessing over uL and uLL depending on platform and architecture.